src/engine/Sim3D.ts

```diff
@@ -152,11 +153,15 @@ export class Sim3D extends EventEmitter {
       this.lastAnimateTime = time;
 
       window.requestAnimationFrame(r);
-      this.updatePhysics(dt);
+      if (!this.stopPhysics) {
+        this.updatePhysics(dt);
+      }
       this.render(dt);
     };
 
     this.isRendering = true;
+    this.stopPhysics = true;
```

Comment:
Did you mean for this to constantly get set here?

Reply:
Haha nope, wrong way round!